src/diffbot.js

```diff
@@ -1,9 +1,13 @@
+const axios = require('axios');
+const { response } = require('express');
 const fetch = require('./lib/fetch-promise');
 
-class Diffbot {
+const sleep = require('util').promisify(setTimeout);
+
+class DiffBot {
   /**
-   * Instantiate a Diffbot
-   * @param {string} token The Diffbot API token to use
+   * Instantiate a DiffBot
+   * @param {string} token The DiffBot API token to use
```

Comment:
```suggestion
   * Instantiate a Diffbot
   * @param {string} token The Diffbot API token to use
```
Keep capitalization consistent